src/containers/Articles/Article.js

```diff
@@ -2,10 +2,12 @@ import React, { Component } from 'react';
 import axios from 'axios';
 import { NavLink } from 'react-router-dom';
 import PropTypes from 'prop-types';
-import classes from '../../CSS/Article.css';
+// import ReactDOM from 'react-dom';
```

Comment:
@AGMETEOR, please take out this commented import. I think it will not be required.

Reply:
@abulojoshua1 let me take it out.

Reply:
Ok.